packages/shared/src/dto/workflows/step-content-issue.enum.ts

```diff
@@ -1,13 +1,16 @@
 export enum StepContentIssueEnum {
-  MISSING_VARIABLE_IN_PAYLOAD = 'MISSING_VARIABLE_IN_PAYLOAD',
   ILLEGAL_VARIABLE_IN_CONTROL_VALUE = 'ILLEGAL_VARIABLE_IN_CONTROL_VALUE',
   MISSING_VALUE = 'MISSING_VALUE',
-  MIN_VALUE = 'MIN_VALUE',
-  INVALID_URL = 'INVALID_URL',
   TIER_LIMIT_EXCEEDED = 'TIER_LIMIT_EXCEEDED',
 }
 
+export enum StepIntegrationIssueEnum {
+  MISSING_INTEGRATION = 'MISSING_INTEGRATION',
+}
+
 export enum StepIssueEnum {
-  STEP_ID_EXISTS = 'STEP_ID_EXISTS',
-  MISSING_REQUIRED_VALUE = 'MISSING_REQUIRED_VALUE',
+  ILLEGAL_VARIABLE_IN_CONTROL_VALUE = 'ILLEGAL_VARIABLE_IN_CONTROL_VALUE',
+  MISSING_VALUE = 'MISSING_VALUE',
+  TIER_LIMIT_EXCEEDED = 'TIER_LIMIT_EXCEEDED',
+  MISSING_INTEGRATION = 'MISSING_INTEGRATION',
```

Comment:
Not DRY here as I couldn't find a way to combine enums without getting ts errors down the line.